README.md

```diff
@@ -386,18 +386,32 @@ public CloudTasksClient getService() throws IOException {
 Long running operations (LROs) are often used for API calls that are expected to
 take a long time to complete (e.g. provisioning a GCE instance or a Dataflow pipeline).
 The initial API call creates an "operation" on the server and returns an operation ID
-to track its progress.
+to track its progress. LRO RPCs will have the suffix `Async` appended to the call name
+(i.e. `clusterControllerClient.createClusterAsync()`)
 
-Our generated gRPC clients provide a nice interface for starting the operation and
+Our generated clients provide a nice interface for starting the operation and
 then waiting for the operation to complete. This is accomplished by returning an
 [`OperationFuture`](https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.longrunning.OperationFuture).
 When you call `get()` on the `OperationFuture` we poll the operation endpoint to
 check on the operation. These polling operations have a default timeout that
 varies from service to service and will throw a `java.util.concurrent.CancellationException`
 with the message: `Task was cancelled.` after that timeout has been reached.
 
-### Configuring LRO Timeouts
+If you receive a `java.util.concurrent.CancellationException`, it does not mean that
```

Comment:
Start a paragraph with a positive sentence, rather than "it does not mean that ..."

Reply:
I feel that " These polling operations have ..." is a beginning of a paragraph for "LRO Timeouts" section.

Reply:
Moved the sections around accordingly. Should be clearer (I hope)